src/app/state/governance/role/services/role.service.ts

```diff
@@ -14,4 +14,12 @@ export class RoleService {
       this.iamService.getENSTypesByOwner(NamespaceType.Role) as Promise<IRole[]>
     );
   }
+
+  getDefinition(namespace: string) {
+    return this.iamService.getRolesDefinition(namespace);
+  }
+
+  getDefinitions(namespaces: string[]) {
+    return this.iamService.cacheClient.getRolesDefinition(namespaces);
+  }
```

Comment:
I think this is a bit confusing to someone reading the code. It's a bit unclear why one is using method in `iamService` and one is going to the `cacheClient` directly. To me, it might be clearer to change `IamService.getRolesDefinition` to accept and return an array and then do:

```suggestion
  getDefinition(namespace: string) {
    return this.iamService.getRolesDefinition([namespace])[0];
  }

  getDefinitions(namespaces: string[]) {
    return this.iamService.getRolesDefinition(namespaces);
  }
```

Reply:
I will remove method from iamService and i will just use directly from cacheClient.
Anyway getRolesDefinition returns an object, where a key is a role namespace.